File: src/main/java/ukalus/Thing.java
package ukalus;

import java.io.Serializable;

/**
 * DOCUMENT ME!
 * 
 * @author pmgremo
 */
public class Thing implements Serializable {

  private static final long serialVersionUID = 3257288011124519993L;
  private Creature owner;

  /**
   * Returns the owner.
   * 
   * @return Creature
   */
  public Creature getOwner() {
    return owner;
  }

  /**
   * Sets the owner.
   * 
   * @param owner
   *          The owner to set
   */
  public void setOwner(Creature owner) {
    this.owner = owner;
  }
}

File: src/main/kotlin/ukalus/Thing.kt
package ukalus

import java.io.Serializable

/**
 * DOCUMENT ME!

 * @author pmgremo
 */
open class Thing : Serializable {
    /**
     * Returns the owner.

     * @return Creature
     */
    /**
     * Sets the owner.

     * @param owner
     * *          The owner to set
     */
    var owner: Creature? = null

    companion object {

        private const val serialVersionUID = 3257288011124519993L
    }
}